test/onnx/test_verify.py

```diff
@@ -75,7 +75,7 @@ def __init__(self):
             def forward(self, x):
                 y = x * x
                 self.param.data.add_(1.0)
-                return y
+                return y, self.param
```

Comment:
@spandantiwari 
This test is designed to verify that model parameters/buffers are not updated in inference mode.
When enabling the constant folding, the model graph no longer contains the initializer and input nodes corresponding to this parameter. The reason is that these nodes are basically leaves in the graph, and are not used.

Eliminating these nodes from the graph, avoids the model parameter update, and hence the negative test fails.
For now, I added the model parameters as part of model outputs, so that the nodes won't get eliminated. The negative test is passing after this change.

Reply:
So we should disable the do_constant_folding in this case.

Reply:
Yeah I guess if we don't want to actually test if model params get updated with constant folding = True, we can set it to False for this test.